frontend/src/components/elements/ExceptionElement/ExceptionElement.tsx

```diff
@@ -32,20 +33,30 @@ class ExceptionElement extends React.PureComponent<Props> {
     const { element, width } = this.props
     const type = element.get("type")
     let message = element.get("message")
-    if (message) {
-      message = `: ${message}`
-    }
     const stackTrace = element.get("stackTrace")
 
+    // On the backend, we use the StreamlitException type for errors that
+    // originate from inside Streamlit. These errors have Markdown-formatted
+    // messages, and so we wrap those messages inside our Markdown renderer.
+    let messageNode: ReactNode
+    if (element.get("messageIsMarkdown")) {
+      messageNode = <StreamlitMarkdown source={message} allowHTML={false} />
+    } else {
+      if (message) {
+        message = `: ${message}`
+      }
+      messageNode = message
+    }
+
     // Put it all together into a nice little html view.
     return (
       <div
         className="alert alert-danger exception stException"
         style={{ width }}
       >
         <div className="message">
-          <strong>{type}</strong>
-          {message}
+          <div className="type">{type}</div>
```

Comment:
Why is the type inside a block element rather than inline? I think inline looks better...